package/src/components/Message/MessageSimple/MessageContent.tsx

```diff
@@ -453,7 +453,8 @@ const areEqual = <
     prevMessage.reply_count === nextMessage.reply_count &&
     prevMessage.status === nextMessage.status &&
     prevMessage.type === nextMessage.type &&
-    prevMessage.text === nextMessage.text;
+    prevMessage.text === nextMessage.text &&
+    prevMessage?.pinned === nextMessage?.pinned;
```

Comment:
```suggestion
    prevMessage.pinned === nextMessage.pinned;
```
It doesn't seem like optional chaining would be necessary here, mind removing it? Or if I'm wrong, why is it needed here and not in other uses of `prevMessage`?

Reply:
<img width="920" alt="Screen Shot 2021-11-30 at 10 47 38" src="https://user-images.githubusercontent.com/27191029/143982414-4bdae31d-14c4-4c99-895c-13146cf3a741.png">
I saw the varible is optional. so Make sure every thing is working i do like this